Source/WebCore/css/DOMCSSRegisterCustomProperty.cpp

```diff
@@ -60,7 +60,7 @@ ExceptionOr<void> DOMCSSRegisterCustomProperty::registerProperty(Document& docum
         auto style = styleResolver->defaultStyleForElement(nullptr);
 
         HashSet<CSSPropertyID> dependencies;
-        CSSPropertyParser::collectParsedCustomPropertyValueDependencies(*syntax, false, dependencies, tokenizer.tokenRange(), strictCSSParserContext());
+        CSSPropertyParser::collectParsedCustomPropertyValueDependencies(*syntax, true, dependencies, tokenizer.tokenRange(), strictCSSParserContext());
```

Comment:
Would be nice to put in an inline comment `/* */` to indicate what this boolean means and switch to an enum class for better readability.